src/config/config.example.js

```diff
@@ -2,5 +2,6 @@ module.exports = {
   sitekey: 'super secret key',
   GENERAL_API_URL: 'http://localhost:8080',
   LOGGING_API_URL: 'http://localhost:8081',
-  MAILER_API_URL: 'http://localhost:8082'
+  MAILER_API_URL: 'http://localhost:8082',
+  RPC_API_URL: 'http://localhost:8083'
```

Comment:
I assume these are for dockers? Are these ports open to each other when running on the local machine (without docker)?

Reply:
yes